lib/node_modules/@stdlib/math/base/special/frexpf/test/test.native.js

```diff
@@ -0,0 +1,212 @@
+	var expected;
+	var x;
+	var f;
+	var i;
+
+	x = medium.x;
+	expected = medium.expected;
+	for ( i = 0; i < x.length; i++ ) {
+		expected[ i ][ 0 ] = f32( expected[ i ][ 0 ] );
+		f = frexpf( x[i] );
+		t.deepEqual( f, expected[ i ], 'returns expected value' );
+	}
+	t.end();
+});
+
+tape( 'the function splits a floating-point number into a normalized fraction and an integer power of two (large `x`)', opts, function test( t ) {
+	var expected;
+	var x;
+	var f;
+	var i;
+
+	x = large.x;
+	expected = large.expected;
+	for ( i = 0; i < x.length; i++ ) {
+		expected[ i ][ 0 ] = f32( expected[ i ][ 0 ] );
+		f = frexpf( x[i] );
+		t.deepEqual( f, expected[ i ], 'returns expected value' );
+	}
+	t.end();
+});
+
+tape( 'the function splits a floating-point number into a normalized fraction and an integer power of two (subnormal `x`)', opts, function test( t ) {
+	var expected;
+	var x;
+	var f;
+	var i;
+
+	x = subnormal.x;
+	expected = subnormal.expected;
+	for ( i = 0; i < x.length; i++ ) {
+		expected[ i ][ 0 ] = f32( expected[ i ][ 0 ] );
+		f = frexpf( x[i] );
+		t.deepEqual( f, expected[ i ], 'returns expected value' );
+	}
+	t.end();
+});
+
+tape( 'the returned normalized fraction and exponent satisfy the relation `x = frac * 2**exp`', opts, function test( t ) {
+	var sign;
+	var frac;
+	var exp;
+	var x;
+	var f;
+	var i;
+
+	for ( i = 0; i < 1000; i++ ) {
+		if ( randu() < 0.5 ) {
+			sign = f32( -1.0 );
+		} else {
+			sign = f32( 1.0 );
+		}
+		frac = f32( randu()*10.0 );
+		exp = roundf( randu()*76.0 ) - 38;
+		x = f32( sign * frac * f32( pow( 10.0, exp ) ) );
+		f = frexpf( x );
+
+		if ( f[ 1 ] > BIAS ) {
+			f = f32( f[ 0 ] * pow( 2.0, BIAS ) * pow( 2.0, f[1]-BIAS ) );
+		} else {
+			f = f32( f[ 0 ] * pow( 2.0, f[ 1 ] ) );
+		}
+		t.strictEqual( f, x, 'returns expected value' );
+	}
+	t.end();
+});
+
+tape( 'the absolute value of the normalized fraction is on the interval `[1/2,1)`', opts, function test( t ) {
+	var sign;
+	var frac;
+	var exp;
+	var x;
+	var f;
+	var i;
+
+	for ( i = 0; i < 1000; i++ ) {
+		if ( randu() < 0.5 ) {
+			sign = f32( -1.0 );
+		} else {
+			sign = f32( 1.0 );
+		}
+		frac = f32( randu()*10.0 );
+		exp = roundf( randu()*74.0 ) - 37;
+		x = f32( sign * frac * f32( pow( 10.0, exp ) ) );
+		f = frexpf( x );
+
+		// Compute the absolute value of the normalized fraction:
+		f = absf( f[ 0 ] );
+
+		t.ok( f >= 0.5 && f < 1.0, 'returns expected value' );
+	}
+	t.end();
+});
+
+tape( 'if provided `+0`, the function returns `[0,0]`', opts, function test( t ) {
+	var f = frexpf( 0.0 );
+	t.deepEqual( f, [ 0.0, 0 ], 'returns expected value' );
+	t.end();
+});
+
+tape( 'if provided `-0`, the function returns `[-0,0]`', opts, function test( t ) {
+	var f = frexpf( -0.0 );
+	t.strictEqual( isNegativeZerof( f[0] ), true, 'returns expected value' );
+	t.deepEqual( f, [-0.0, 0], 'returns expected value' );
```

Comment:
Here and below, can we please add spaces around square brackets like in L184? I know it's the same in `frexp`, but would be better to change there as well. No good reason for the inconsistency or to deviate from our convention of including spaces.